### PR TITLE
Feature/getcode implementation

### PR DESCRIPTION
**Pull request by flexxnn**
Added method eth.getCode with unit tests
Updated docs

### CODE REVIEW

File: test/testETHAPI.js [hunk 1]
```diff
@@ -275,4 +278,36 @@ describe('ETH API', function() {
...
 
+  describe('getCode', function() {
+    it('contract with empty code', function(done) {
+      eth.getCode({
+        rpcaddr,
+        rpcport,
+        address: addressWithoutContract,
+      }).then((result) => {
+        result.should.be.a.String()
+        if (result !== '0x' && result !== '0x0') {
```
Comment (IViol) [inline test/testETHAPI.js]: better to wrap this check in function and reuse it in both tests
example of function:
```
function checkZeroResult(value) {
  return ((value === '0x') || (value === '0x0'))
}
```

Comment (Romanior) [reply test/testETHAPI.js]: still what is `0x` or `0x0` not clear

File: test/testETHAPI.js [hunk 2]
```diff
@@ -19,6 +19,9 @@ const address = testParams.accounts[0]
 const to = testParams.accounts[1]
 const value = new BigNumber(1, 10)
 
+const addressWithoutContract = '0x0000000000000000000000000000000000000000'
+const addressWithContract = testParams.contracts.JNTController
```
Comment (Romanior) [inline test/testETHAPI.js]: need to move const 0x 0x0 here as well.

File: test/testETHAPI.js [hunk 3]
```diff
@@ -275,4 +278,36 @@ describe('ETH API', function() {
     })
   })
 
+  describe('getCode', function() {
+    it('contract with empty code', function(done) {
```
Comment (Romanior) [inline test/testETHAPI.js]: 'contract with empty code should be a string' or something, just write what to expect from the test

File: test/testETHAPI.js [hunk 4]
```diff
@@ -275,4 +278,36 @@ describe('ETH API', function() {
...
+        if (result !== '0x' && result !== '0x0') {
+          throw new Error('Result should be 0x or 0x0')
+        }
+
+        done()
+      }).catch(done)
+    })
+
+    it('contract with existing code', function(done) {
+      eth.getCode({
```
Comment (Romanior) [inline test/testETHAPI.js]: probably with `async await` this would be cleaner to write.

File: test/testETHAPI.js (hunk 1 above)
Comment (Romanior) [inline test/testETHAPI.js]: does not look good in test this structure, better
```
const hasResult = (result === '0x' || result === '0x0')
expect(hasResult).to.be.true;
```

**Pull request merged**